index.html

```diff
@@ -275,9 +276,14 @@ <h2 class="contact-title headline--container">Contact me</h2>
               rows="10"
               maxlength="500"
```

Comment:
- [ ] [OPTIONAL] I think that it would be nice to make `textarea` required as well as other input fields 😉

Reply:
Thank you very much. 
Noted. I will correct it accordingly.